lib/code-path-analysis/debug-helpers.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
      * @param {boolean} leaving - A flag whether or not it's leaving
      * @returns {void}
      */
-    dumpState: !debug.enabled ? debug : /* istanbul ignore next */ function(node, state, leaving) {
+    dumpState: debug.enabled ? /* istanbul ignore next */ function(node, state, leaving) {
```

Comment:
This would be an example where I don't think the rule is always useful. Usually it makes sense to do consequent then alternate for a positive test case. But personally I like multi-line expressions to be at the end of a line where possible, so a developer doesn't have to look down five lines to find the rest of the ternary expression.

Are we absolutely sure we want to turn this on full bore? Maybe could we have it apply to if statements only for now (if the rule allows that)?